src/components/PostPageDetail/index.js

```diff
@@ -582,6 +586,11 @@ const PostPageDetailIdComponent = (props) => {
             isBackButton={true}
             source={SOURCE_PDP}
             height={getHeightHeader()}
+            isFollow={item?.is_following_target || parentData?.is_following_target}
+            onPressFollUnFoll={() => {
+              followUnfollow(item);
+              setItem({...item, is_following_target: !item?.is_following_target});
+            }}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

This change introduces a follow/unfollow feature. It's important to ensure that the `followUnfollow` function correctly updates the server-side state of whether the user is following the target or not. Also, make sure that the local state update (`setItem`) doesn't lead to inconsistencies with the server-side state in case the `followUnfollow` request fails.
```diff
+ isFollow={item?.is_following_target || parentData?.is_following_target}
+ onPressFollUnFoll={() => {
+   followUnfollow(item);
+   setItem({...item, is_following_target: !item?.is_following_target});
+ }}
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->